user/libyang/update.py

```diff
@@ -0,0 +1 @@
+pattern = r"2\.1\.[\d.]+"
```

Comment:
assuming we don't want 2.2+ because of the sover bump?

Reply:
yeap, FRR needs this one